File: RPC09-17/talkingp.java
import java.io.*;
import java.util.*;

public class talkingp
{
	public static void main(String []args)
	{
		Scanner in = new Scanner(new BufferedReader(new InputStreamReader(System.in)));
		PrintWriter wff = new PrintWriter(System.out);
		int n = Integer.parseInt(in.nextLine());
		String jer[] = "apa epe ipi opo upu".split(" ");
		String voc[] = "a e i o u".split(" ");
		while(n>0)
		{
			String m = in.nextLine();
			for(int i = 0; i<jer.length; i++)
			{
				m = m.replace(voc[i],jer[i]);
			}
			wff.println(m);
			n--;
		}
		wff.flush();
		
	}
}

File: RPC09-17/talkingp.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
	ios_base::sync_with_stdio(0);cin.tie(0);
	string n;
	getline(cin,n);
	int x = atoi(n.c_str());
	while(x>0)
	{
		string s="";
		string res = "";
		getline(cin,s);
		for(int i = 0; i < s.length(); i++)
		{
			if(s[i] == 'a')
			{
				res= res + "apa";	
			}else if(s[i] == 'e')
			{
				res= res + "epe";	
			}else if(s[i] == 'i')
			{
				res= res +  "ipi";
			}else if(s[i] == 'o')
			{
				res= res + "opo";	
			}else if(s[i] == 'u')
			{
				res= res +  "upu";
			}else{
				res= res + s[i];
			} 
		}
		cout << res << endl;
		x--;
	}
	return 0;
}